app/build.gradle

```diff
@@ -170,6 +170,7 @@ dependencies {
     testImplementation "org.robolectric:robolectric:4.3.1"
     testImplementation "org.mockito:mockito-inline:3.2.4"
     testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0"
+    testImplementation "org.conscrypt:conscrypt-openjdk-uber:2.0.0"
```

Comment:
this is so Conscrypt can be initialized when running tests on the JVM and not on Android

Reply:
I'm not sure I like it more than stub...

Reply:
I replaced it with a better solution